x/market/internal/keeper/oracle_hooks.go

```diff
@@ -0,0 +1,32 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	core "github.com/terra-project/core/types"
+	"github.com/terra-project/core/x/market/internal/types"
+)
+
+// AfterValidatorBonded register voting info for a validator
+func (k Keeper) AfterPriceUpdated(ctx sdk.Context, denom string, price sdk.Dec) {
+	if denom == core.MicroSDRDenom {
```

Comment:
Add comment as to why SDR has an exception

Reply:
Removed